examples/profile.html

```diff
@@ -0,0 +1,307 @@
+                    profileData.images.forEach(function(currImg) {
+                        if (!currImg.resolution || !currImg.imageUri) {
+                            return;
+                        }
+
+                        let currRes = parseInt(currImg.resolution.substr(1), 10);
+                        if (currRes && !isNaN(currRes) && currRes > bestImage.size && currRes <= 400) {
+                            bestImage.size = currRes;
+                            bestImage.uri = currImg.imageUri;
+                        }
+                    });
+                }
+
+                let placeholderPicEl = profileRootEl.querySelector('.placeholder-pic');
+                let picEl = profileRootEl.querySelector('.profile-pic');
+                if (bestImage.size) {
+                    setHidden(placeholderPicEl, true);
+
+                    picEl.setAttribute('src', bestImage.uri);
+                    if (profileData.name) {
+                        picEl.setAttribute('aria-label', 'Profile picture for ' + profileData.name);
+                    }
+                    setHidden(picEl, false);
+                } else {
+                    setHidden(placeholderPicEl, false);
+                    setHidden(picEl, true);
+                }
+
+                profileRootEl.querySelector('.base-info .name').innerText = profileData.name || 'Unknown User';
+
+                let titleEl = profileRootEl.querySelector('.base-info .title');
+                titleEl.innerText = profileData.title || '';
+                setHidden(titleEl, !profileData.title);
+
+                let deptEl = profileRootEl.querySelector('.base-info .department');
+                deptEl.innerText = profileData.department || '';
+                setHidden(deptEl, !profileData.department);
+            }
+
+            /**
+             * Determine the embedding PureCloud environment seeded on the query string or
+             * being returned through the OAuth2 Implicit grant state hash param.
+             *
+             * @returns A string indicating the embedding PC env (e.g. mypurecloud.com, mypurecloud.jp); otherwise, null.
+             */
+            function getEmbeddingPCEnv() {
+              let result = null;
+
+              if (window.location.hash && window.location.hash.indexOf('access_token') >= 0) {
+                let oauthParams = extractParams(window.location.hash.substring(1));
+                if (oauthParams && oauthParams.access_token && oauthParams.state) {
+                  // OAuth2 spec dictates this encoding
+                  // See: https://tools.ietf.org/html/rfc6749#appendix-B
+                  let stateSearch = unescape(oauthParams.state);
+                  result = extractParams(stateSearch).pcEnvironment;
+                }
+              }
+
+              if (!result && window.location.search) {
+                result = extractParams(window.location.search.substring(1)).pcEnvironment;
```

Comment:
This will technically result in undefined.  Need to modify as the doc states it will return as null.